.circleci/config.yml

```diff
@@ -22,21 +31,202 @@ anchors:
       # In Feb 2017, this was set to 3G. But in Feb 2019 (RW-2194) we started seeing OOM errors,
       # so we bumped this down further to 2G.
       JAVA_TOOL_OPTIONS: -Xmx2g
+      # https://docs.gradle.org/6.3/userguide/gradle_daemon.html#sec:disabling_the_daemon
+      GRADLE_OPTS: "-Dorg.gradle.daemon=false"
       TERM: dumb
-    update_git_submodules: &update_git_submodules
-      name: Update Git Submodules
-      command: git submodule update --init --recursive
-jobs:
-  api-build-test:
-    <<: *java_defaults
+
+  # See https://circleci.com/docs/2.0/workflows/#git-tag-job-execution
+  # Job runs for master branch only
+  filter_only_master_branch: &filter_only_master_branch
+    tags:
+      ignore: /.*/
+    branches:
+      only: master
+
+  # Job runs for no branch and only for tags starting with ‘v’
+  filter_only_release_tags: &filter_only_release_tags
+    branches:
+      ignore: /.*/
+    tags:
+      # Workbench release tag format example: v5-3-rc1
+      only: /^v[0-9]+-[0-9]+-rc[0-9]+$/
```

Comment:
regex short explanation:
`^` asserts position at start of a line.
`v` matches the character v literally (case sensitive).
`[0-9]+` match a single numerical digit (the `+` quantifier — matches between one and unlimited times, as many times as possible).
`-` matches the character `-` literally (case sensitive).
`rc` matches the character `rc` literally.
`$` asserts position at the end of a line.

Reply:
Awesome. Maybe put this in the comments? I doubt anyone will find this conversation again.